src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OtlpLogRecordTransformer.cs

```diff
@@ -164,12 +165,21 @@ internal OtlpLogs.LogRecord ToOtlpLog(LogRecord logRecord)
                     if (attribute.Key.Equals("{OriginalFormat}") && !bodyPopulatedFromFormattedMessage)
                     {
                         otlpLogRecord.Body = new OtlpCommon.AnyValue { StringValue = attribute.Value as string };
+                        useTemplateFromExtensionMethod = true;
                     }
                     else if (OtlpKeyValueTransformer.Instance.TryTransformTag(attribute, out var result, attributeValueLengthLimit))
                     {
                         AddAttribute(otlpLogRecord, result, attributeCountLimit);
                     }
                 }
+
+                // Supports Body set directly on LogRecord for the Logs Bridge API.
+                if (!useTemplateFromExtensionMethod && !bodyPopulatedFromFormattedMessage && logRecord.Body != null)
```

Comment:
I would probably rename `useTemplateFromExtensionMethod` -> `bodyPopulatedFromMessageTemplate` but why not just...

```csharp
if (otlpLogRecord.Body == null && logRecord.Body != null)
{
   otlpLogRecord.Body = new OtlpCommon.AnyValue { StringValue = logRecord.Body };
}
```